src/Agent/NewRelic/Agent/Core/Api/TransactionBridgeApi.cs

```diff
@@ -37,50 +37,31 @@ public TransactionBridgeApi(ITransaction transaction, IApiSupportabilityMetricCo
             _configSvc = configSvc;
         }
 
+        [Obsolete("This method has been stubbed out")]
```

Comment:
Similar comment about being explicit - in non-programmer-speak

Reply:
Done